include/fdreadoutlibs/wibeth/WIBEthFrameProcessor.hpp

```diff
@@ -145,7 +145,7 @@ class WIBEthFrameProcessor : public datahandlinglibs::TaskRawDataProcessorModel<
   //std::array<uint, swtpg_wibeth::NUM_REGISTERS_PER_FRAME * swtpg_wibeth::SAMPLES_PER_REGISTER> m_register_channels;
   std::vector<std::pair<int16_t, int16_t>> m_channel_plane_numbers;
 
-  std::shared_ptr<iomanager::SenderConcept<trigger::TriggerPrimitiveTypeAdapter>> m_tp_sink;
+  std::shared_ptr<iomanager::SenderConcept<trigger::TriggerPrimitiveTypeAdapter>> m_tp_sink[3];
```

Comment:
Will this always be 3? Should we use std::array or std::vector?

Reply:
this may change if we decide for different subdivisions.